### ISSUE TITLE
Result type

### ISSUE THREAD

**Issue by waldemarhorwat** [js-choi/proposal-bigint-bit-length#2]
### Should the result type should be a Number or a BigInt?

We should be consistent with how we treat lengths in the other BigInt operators. In particular, the answer should match the inverse operation: given a bit length, produce a BigInt of that size. That inverse operation is either `1n << length` or `1n << length-1n` depending on whether we choose the off-by-one version or not, but in either case it wants the length represented as a BigInt. So that points towards returning a BigInt.

**Comment by gameroman** [comment]
I agree that for math operations returning a `BigInt` would be more useful

At the same time for cryptography, returning a `Number` could be more useful

Another useful feature would be a `BigInt.byteLength`, also for cryptography, but probably as a separate proposal
